src/main/java/com/agileavengers/icuconnectbackend/service/implementation/UserService.java

```diff
@@ -121,7 +125,7 @@ private User updateFields(User user, UserDetailDto update) {
             user.setEmail(update.getEmail());
         }
         if (update.getStudyArea() != null) {
-            user.setStudyArea(update.getStudyArea());
+            user.setStudyArea(studyAreaService.getOrCreateStudyArea(update.getStudyArea()));
```

Comment:
This implies that studyArea can be any string right? Shouldn't we use enums or smth along those lines to restrict study areas to the ones that exist at Uni?

Reply:
yes this is currently the case. the same for instructors. I agree, that this leaves the user too many possibilities, however I think this is out of the scope for now.